iroh-sync/src/store/memory.rs

```diff
@@ -108,8 +107,8 @@ impl super::Store for Store {
             .into_iter())
     }
 
-    fn import_namespace(&self, namespace: Namespace) -> Result<()> {
-        self.namespaces.write().insert(namespace.id(), namespace);
+    fn import_namespace(&self, capability: Capability) -> Result<()> {
+        self.namespaces.write().insert(capability.id(), capability);
```

Comment:
I think this would delete the secret key when importing a read capability for an existing namespace, which is  likely not the desired outcome.

Reply:
Fixed